packs/data/equipment.db/orc-warmask.json

```diff
@@ -0,0 +1,87 @@
+        },
+        "equippedBulk": {
+            "value": ""
+        },
+        "hardness": 0,
+        "hp": {
+            "brokenThreshold": 0,
+            "max": 0,
+            "value": 0
+        },
+        "level": {
+            "value": 0
+        },
+        "negateBulk": {
+            "value": "0"
+        },
+        "preciousMaterial": {
+            "value": null
+        },
+        "preciousMaterialGrade": {
+            "value": null
+        },
+        "price": {
+            "value": {}
+        },
+        "quantity": 1,
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrcWarmask.Gods",
+                        "value": "religion"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrcWarmask.Land",
+                        "value": "nature"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrcWarmask.Magic",
+                        "value": "arcana"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrcWarmask.Unknown",
+                        "value": "occultism"
+                    }
+                ],
+                "flag": "tradition",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Orc.OrcWarmask.Prompt"
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "{item|flags.pf2e.rulesSelections.tradition}",
+                "type": "item",
+                "value": 1
+            }
```

Comment:
This is not a default feature, you need to do extra things to activate this benefit.

Reply:
It seemed like a very specific thing to add a toggle for. It's an invested item with no benefit until you spend the money.